### PR TITLE
Research/parallel-tx/main

### PR DESCRIPTION
**Pull request by julianlen**

### CODE REVIEW

File: rskj-core/src/main/java/co/rsk/core/bc/ParallelizeTransactionHandler.java [hunk 1]
```diff
@@ -0,0 +1,259 @@
...
+
+import java.util.*;
+
+public class ParallelizeTransactionHandler {
+    private final HashMap<ByteArrayWrapper, TransactionSublist> sublistByWrittenKey;
+    private final HashMap<ByteArrayWrapper, Set<TransactionSublist>> sublistsByReadKey;
+    private final Map<RskAddress, TransactionSublist> sublistBySender;
+    private final ArrayList<TransactionSublist> sublists;
+
+    public ParallelizeTransactionHandler(short sublists, long sublistGasLimit) {
```
Comment (ilanolkies) [inline rskj-core/src/main/java/co/rsk/core/bc/ParallelizeTransactionHandler.java:34]: `sublists` has the same name as `this.sublists`

```suggestion
    public ParallelizeTransactionHandler(short parallelSublistsAmount, long sublistGasLimit) {
```

**Review by ilanolkies — CHANGES_REQUESTED** [review]

**Comment by Vovchyk** [comment]
pipeline:run

**Comment by Vovchyk** [comment]
pipeline:run

**Comment by ilanolkies** [comment]
Closing in favor of #1930

**Pull request closed without merging**